client/translations/admin-clients.en.yml

```diff
@@ -4,7 +4,7 @@ clientSearch:
   searchBy: Search by client surname or case number
   clientTable:
     heading: Client list
-    results: Found %numberOfClients% clients jim
```

Comment:
Hahahaha how long has that been in there? 😆

Reply:
The shame!!!! It was probably a different Jim... not me, nothing to see here... Since I did that massive `raw` removal PR. So er... quite a while ;)